src/main/kotlin/com/server/dpmcore/security/oauth/token/JwtTokenProvider.kt

```
@@ -50,22 +52,30 @@ class JwtTokenProvider(
-    fun validateToken(token: String?): Boolean {
-        return try {
-            Jwts.parser().verifyWith(getSigningKey()).build().parseSignedClaims(token)
+    fun validateToken(token: String?): Boolean =
+        try {
+            Jwts
+                .parser()
+                .verifyWith(getSigningKey())
+                .build()
+                .parseSignedClaims(token)
             true
         } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)